src/WinRT.Runtime/Projections/INotifyDataErrorInfo.net5.cs

```diff
@@ -33,6 +33,13 @@ public static unsafe bool get_HasErrors(IObjectReference obj)
 
         public static unsafe global::System.Collections.IEnumerable GetErrors(IObjectReference obj, string propertyName)
         {
+            if (!FeatureSwitches.IsDynamicObjectsSupportEnabled)
```

Comment:
EnableIDynamicInterfaceCastableSupport?

Reply:
Oops, fixed in d3ffb5c9620ca9727ca6ed47987e96dc1c233288, thank you! Also renamed the one for dynamic objects to make it consistent with the other names, that's why I tripped up and had accidentally used it here 😅